### PR TITLE
Limit and update required Azure packages

### PR DESCRIPTION
**Pull request by tiloc**
Limit and update required Azure packages. See issue #680

### CODE REVIEW

**Comment by dazzag24** [comment]
+1 for this.  Requiring the base azure package pulls in far too many azure sub modules that are not needed. This is making our CI tests slower than they need to be.
Thanks

File: setup.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def read(filename):
     python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
     install_requires=['Django>=1.11'],
     extras_require={
-        'azure': ['azure>=3.0.0', 'azure-storage-blob>=1.3.1'],
+        'azure': ['azure-common', 'azure-storage-blob>=1.3.1'],
```
Comment (nitely) [inline setup.py]: `azure-common` should be removed. `azure-storage-blob` already depends on it (so it will install it)

Comment (akshaybabloo) [reply setup.py]: True

Comment (jschneier) [reply setup.py]: I took this into consideration in #684

**Pull request closed without merging**